integration/it-integration/src/main/java/se/inera/intyg/rehabstod/integration/it/dto/SickLeavesRequestDTO.java

```diff
@@ -20,69 +20,17 @@
 package se.inera.intyg.rehabstod.integration.it.dto;
 
 import java.util.List;
+import lombok.Data;
+import se.inera.intyg.infra.sjukfall.dto.DiagnosKapitel;
 
+@Data
 public class SickLeavesRequestDTO {
     private String unitId;
     private String careUnitId;
     private List<String> doctorIds;
+    private List<DiagnosKapitel> diagnosisChapters;
     private int maxCertificateGap;
     private int maxDaysSinceSickLeaveCompleted;
     private int toSickLeaveLength;
```

Comment:
Should we make this to an Integer so we can handle null values, or will it never be null?

Reply:
As it is now it can't be null since the limits are 1 and 365. They haven't decided on the final behavior for the filter so I think we can leave it as it is now and make the changes when they decide.

Reply:
Alright, sounds good!